cpp/include/raft/core/mdspan.hpp

```diff
@@ -33,6 +33,9 @@ template <typename ElementType,
           typename AccessorPolicy = std::experimental::default_accessor<ElementType>>
 using mdspan = std::experimental::mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy>;
 
+template <class IndexType, std::size_t Rank>
+using dextents = std::experimental::dextents<IndexType, Rank>;
```

Comment:
This seems to be only used in tests, is it? If so, do we need to re-export this definition?